src/frequenz/sdk/microgrid/component/_component_data.py

```diff
@@ -137,6 +142,9 @@ class BatteryData(ComponentData):
     power_lower_bound: float
     power_upper_bound: float
     temperature_max: float
+    _relay_state: battery_pb.RelayState.ValueType
+    _component_state: battery_pb.ComponentState.ValueType
+    _errors: List[battery_pb.Error]
```

Comment:
So here, like:
```py
    _relay_state: battery_pb.RelayState.ValueType
    """State of the battery relay."""
    _component_state: battery_pb.ComponentState.ValueType
    """State of the battery."""
    _errors: List[battery_pb.Error]
    """List of errors in protobuf struct."""
```

With dataclasses it will say they are class attributes, but it is also marked as a dataclass so it is still easy for users to know they are really instance attributes.

Reply:
Done (last commit).